microsoft-edge/toc.yml

```
@@ -496,11 +496,11 @@
 # Performance tool ------------------------------------------------------------
             - name: Performance tool
               items:
-                - name: Get started analyzing runtime performance
+                - name: Introduction to the Performance tool
                   href: devtools-guide-chromium/evaluate-performance/index.md
                   displayName: Performance tool
 
-                - name: Analyze runtime performance
+                - name: Troubleshooting common performance issues
                   href: devtools-guide-chromium/rendering-tools/index.md
                   displayName: Performance tool
```

```suggestion
                  displayName: Performance tool, Analyze runtime performance  # old title
```
make this change (adding an alt index lookup string) if ppl might look up old title in upper left multipurpose text box